src/app/clusters/mode-select-server/mode-select-server.cpp

```diff
@@ -28,12 +28,22 @@
 #include <app/clusters/mode-select-server/supported-modes-manager.h>
 #include <app/util/af.h>
 #include <app/util/attribute-storage.h>
+#include <app/util/error-mapping.h>
+#include <app/util/util.h>
 #include <lib/support/CodeUtils.h>
 
 using namespace std;
 using namespace chip;
 using namespace chip::app;
 using namespace chip::app::Clusters;
+using namespace chip::app::Clusters::ModeSelect;
+using namespace chip::Protocols::InteractionModel;
+
+#ifndef IGNORE_MODE_SELECT_CLUSTER_START_UP_MODE
+static bool areStartUpModeServerAttributesNonVolatile(EndpointId endpoint);
+#endif
+
+Protocols::InteractionModel::Status verifyModeValue(const EndpointId endpointId, const uint8_t newMode);
```

Comment:
Should be static or in anonymous namespace, no?

Reply:
Updated to static in new push.